$OpenBSD: patch-src_gpm-backlight-helper_c,v 1.1 2011/05/24 18:14:52 ajacoutot Exp $

XXX make gnome-power-backlight-helper do nothing and return 0.

--- src/gpm-backlight-helper.c.orig	Fri Aug  6 15:50:47 2010
+++ src/gpm-backlight-helper.c	Tue May 24 19:32:32 2011
@@ -141,6 +141,7 @@ out:
 gint
 main (gint argc, gchar *argv[])
 {
+#ifndef __OpenBSD__
 	GOptionContext *context;
 	gint uid;
 	gint euid;
@@ -276,5 +277,9 @@ out:
 	g_free (filename_file);
 	g_free (contents);
 	return retval;
+#else
+	g_print ("This program does nothing on OpenBSD (no support for sysfs(2)).\n");
+	return 0;
+#endif
 }
 
